### PR TITLE
(576) Add exclusion zones page

### PR DESCRIPTION
**Pull request by libuk**
# Changes in this PR
* Update question copy for the exclusion zones page
* Adds ExclusionZones page to the apply journey

### Before
image

### After
image

# Release checklist

As part of our continuous deployment strategy we must ensure that this work is
ready to be released at any point. Before merging to `main` we must first
confirm:

## Pre merge checklist

- [ ] Are any changes required to the e2e tests?
- [ ] If you've added a new route, have you added a new
  `auditEvent`? (see `server/routes/apply.ts` for examples)
- [ ] Are there environment variables or other infrastructure configuration which needs to be included in this release?
- [ ] Are there any data migrations required. Automatic or manual?
- [ ] Does this rely on changes being deployed to the CAS API?

## Post merge checklist

Once we've merged it will be auto-deployed to the dev environment.

[Find the build-and-deploy job in CircleCI](https://app.circleci.com/pipelines/github/ministryofjustice/hmpps-community-accommodation-tier-2-ui).

Should a release fail at any step, you as the author should now lead the work to
fix it as soon as possible.

### CODE REVIEW

**Comment by libuk** [comment]
[Area - Exclusion zones](https://trello.com/c/oi6dm43S/576-area-exclusion-zones)

File: integration_tests/pages/apply/taskListPage.ts [hunk 1]
```diff
@@ -32,7 +32,7 @@ export default class TaskListPage extends Page {
   }
 
   shouldShowTaskStatus = (task: string, status: string): void => {
-    cy.get(`#${task}-status`).should('contain', status)
+    cy.get(`#${task}-status`).debug().should('contain', status)
```
Comment (patrickjfl) [inline integration_tests/pages/apply/taskListPage.ts:35]: is this intentional?

File: integration_tests/tests/apply/area_and_funding/area_information/exclusion_zones.cy.ts [hunk 2]
```diff
@@ -0,0 +1,91 @@
...
+  it('navigates to the next page', function test() {
+    // So that the status of the task will be complete we set application.data
+    // to the full set
+    cy.fixture('applicationData.json').then(applicationData => {
+      const answered = {
+        ...this.application,
+        data: applicationData,
+      }
+      cy.task('stubApplicationGet', { application: answered })
+    })
```
Comment (patrickjfl) [inline integration_tests/tests/apply/area_and_funding/area_information/exclusion_zones.cy.ts:78]: looks like you've done the set up but you're missing the test to check that the task status is complete

Comment (patrickjfl) [reply integration_tests/tests/apply/area_and_funding/area_information/exclusion_zones.cy.ts:78]: wait, sorry, this is not the final page in the task, right? In which case the set up is unnecessary

**Review by patrickjfl — APPROVED** [review]
nice work, lgtm 👍

File: integration_tests/pages/apply/taskListPage.ts (hunk 1 above)
Comment (libuk) [reply integration_tests/pages/apply/taskListPage.ts:35]: Not sure how this got here, looks like some funkiness with git history, as this was fixed in an earlier PR. I'll drop the commit.

**Pull request merged**